zone/attack.cpp

```diff
@@ -1654,7 +1668,7 @@ bool Client::Attack(Mob* other, int Hand, bool bRiposte, bool IsStrikethrough, b
 	///////////////////////////////////////////////////////////
 	other->Damage(this, my_hit.damage_done, SPELL_UNKNOWN, my_hit.skill, true, -1, false, m_specialattacks);
 
-	if (IsDead()) {
+	if (CastToClient()->IsDead() || GetHP() < 0) {
```

Comment:
What's with the behavior change / check here with GetHP() < 0 ?

Reply:
Changed to (IsBot() && GetAppearance() == eaDead), Bots don't have a IsDead Method, so this appears to be the correct way to check.